CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ All notable changes to this project will be documented in this file.
 -   #1665 : Add C and Fortran support for returning lists from functions.
 -   #1689 : Add C and Fortran support for list method `append()`.
 -   #1876 : Add C support for indexing lists.
--   #1690 : Add C support for list method `pop()`.
+-   #1690 : Add C and Fortran support for list method `pop()`.
```

Comment:
Didn't we agree to use `Add support` when a functionality is fully supported?

Reply:
Not quite, we agreed to use `Add support` when support for all languages was added. In this case Python support was already available in the previous release.